CHANGELOG.md

```diff
@@ -5,7 +5,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [2.0.0] - 2024-04-03
```

Comment:
The changelog should not be updated with a version until that version is generated. These notes should remain in the `Unreleased` section. Part of the release process is to move the notes from the unrelease section to whatever version gets created.

In this case, this would need to be a `3.0.0` version since its got breaking changes.

Reply:
Fixed.